credentials/src/main/kotlin/web5/sdk/credentials/StatusListCredential.kt

```diff
@@ -1,6 +1,5 @@
 package web5.sdk.credentials
 
-import com.danubetech.verifiablecredentials.CredentialSubject
```

Comment:
could you also remove danubetech dependency from build.gradle.kts?

Reply:
I can't do this yet because I need to create a VerifiablePresentation Data model to fully remove danubetech (coming soon)

Reply:
idk if I should include it in this pr, But I kind of want to another, because this one is already big